yaml/xyz/openbmc_project/Provisioning/Provisioning.interface.yaml

```diff
@@ -0,0 +1,66 @@
+description: >
+    Interface representing the provisioning state and peer connectivity status
+    of the BMC. It provides properties to indicate whether the BMC has been
+    provisioned and the current connection state with the peer BMC, along with a
+    method to initiate peer provisioning and a signal to report its result.
+
+properties:
+    - name: Provisioned
+      type: boolean
+      flags:
+          - readonly
+      description: >
+          True means the BMC is in a provisioned state.
```

Comment:
Suggestion: As mentioned early, looks it not yet addressed yet but I will leave it to you.

`True means this BMC is in a provisioned state.`